cmd/state/main.go

```diff
@@ -150,6 +150,17 @@ func run(args []string, isInteractive bool, cfg *config.Instance, out output.Out
 
 	svcmodel := model.NewSvcModel(svcPort)
 
+	// Give the logger a way to fetch the state-svc log tail. This cannot be done inside the logger
+	// package itself because importing pkg/platform/model creates an import cycle.
+	logging.SetSvcTailProvider(func() string {
+		ctx, cancel := context.WithTimeout(context.Background(), model.SvcTimeoutMinimal)
+		defer cancel()
+		if tail, err := svcmodel.FetchLogTail(ctx); err == nil {
+			return tail
+		}
+		return ""
```

Comment:
It might be worth displaying the error if one is encountered: `return fmt.Sprintf("svc log tail not found: %v", err)`.

Reply:
I'm not sure about that. The user is already going to get a display that something went wrong, so this would just be added noise that's more confusing to them.

Reply:
Oh wait, I see what you mean. This formatted error will be uploaded to rollbar. Got it. That's a good idea.